CoordgenFragmentBuilder.h

```diff
@@ -63,11 +64,15 @@ class EXPORT_COORDGEN CoordgenFragmentBuilder
     }
 
     /* set precision of the calculations. Higher precisions settings result
-     better
-     quality but slower
-     calculations */
+     better quality but slower calculations */
     void setPrecision(float f) { m_macrocycleBuilder.setPrecision(f); }
 
+    /* Set a timeout (in seconds) for construction of each fragment.
```

Comment:
The "construction of each fragment" part confuses me, it makes me think that we run the check for each fragment.. it's to account for the fact that we might recurse on macrocycles, right? how about something like a "a timeout for clash minimization (might be run several times on hard macrocyles, up to once for each macrocycles)"

Reply:
I basically didn't want to have it _too_ focused on the implementation, while still giving callers some sense of what it is doing.

Your suggestino fits that really nicely. Updating.